locator/tablets.cc

```diff
@@ -981,3 +1051,44 @@ auto fmt::formatter<locator::tablet_metadata_change_hint>::format(const locator:
     }
     return fmt::format_to(out, "\n}}");
 }
+
+auto fmt::formatter<locator::repair_scheduler_config>::format(const locator::repair_scheduler_config& config, fmt::format_context& ctx) const
+        -> decltype(ctx.out()) {
+    std::map<sstring, sstring> ret{
+        {"auto_repair_enabled", config.auto_repair_enabled ? "true" : "false"},
+        {"auto_repair_threshold", std::to_string(config.auto_repair_threshold.count())},
```

Comment:
See below.  I suggest to rename this to `auto_repair_interval_in_seconds`

Reply:
This looks too long in the cql query results. We need to document the meaning of this config anyway. This short name looks nicer and is more consistent with the rest of the other options.

Reply:
But as a user, I wouldn't understand what `auto_repair_threshold = 60` means without going to the docs.

Reply:
Users will not understand what this config mean exactly even if  the name has the word seconds in the name without looking at the docs.